index.html

```diff
@@ -117,6 +117,44 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+         <!-- ________ HareerSakhi's  Contributor card START ________  -->
+         <div class="card">
+          <p class="name">HareerSakhi</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/HareerSakhi" target="_blank">HareerSakhi</a>
+          </p>
+          <p class="about">
+            <ul>
+            <li>HUGE Harry Potter fan</li>
+            <li>Hopeful web and app developer</li>
+            <li>True science lover</li> 
+            <li>A football-watching frenzy</li>
+          </ul>
```

Comment:
Hello @HareerSakhi, thank you for contributing. One change needs to be made. It is best practice that if you are going to add an unordered list as part of a paragraph element to close the paragraph tag then add your unordered list. Here is an example
![image](https://user-images.githubusercontent.com/48189365/125220658-1e392c80-e295-11eb-80e5-c952ec5ebd56.png)
Once you have made this change, run `npm run prettier-html` and make sure it passes the styling guidelines, and then push your commit to this PR and we will get your card added. Let us know if you have any questions. Take care 🚀